packages/i18n/src/components/NestedLanguageProvider.tsx

```diff
@@ -25,7 +25,7 @@ const NestedLanguageProvider = ({
   const { messages: fallbackMessages } = useIntl();
   const [searchParams] = useSearchParams();
   const locale = searchParams.get("locale");
-  const localeMessages = messages.get(locale || "");
+  const localeMessages = messages.get(locale ?? "");
```

Comment:
Pre-existing: There's no "" key, right?  This will fail anyway?

Reply:
Updated in [ignore boolean primitives](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11651/commits/ccb9ba3ec73672fea328bb97fecdb8b1c1db5b91)